pep-9999.rst

```diff
@@ -357,7 +357,10 @@ specified in ``__all__`` are imported::
   public_attr: int
   _private_looking_public_attr: int
   private_attr: int
-  
+
+A type stub should contain an ``__all__`` variable if and only if the
+implementation has one too, and it should be identical to the implementation.
```

Comment:
Something to consider: what if `__all__` is dynamically generated at runtime? We should probably say that it should be equal to the actual runtime value, even if at runtime it uses concatenation or manual list manipulation to construct `__all__`.

Reply:
Good point, I will change the wording to that effect. I also move this paragraph to the "What to include" section, where we also already have a paragraph about `__all__`.

Reply:
Update: It existed in the style guide section. I am shuffling all of this around a bit.